services/web-app/pages/libraries/[host]/[org]/[repo]/[library]/[ref]/index.js

```diff
@@ -246,6 +262,25 @@ export const getServerSideProps = async ({ params }) => {
     }
   }
 
+  const relatedLibs = []
+  for (const [slug, libraryParams] of Object.entries(libraryAllowList)) {
+    if (libraryParams.group === libraryData.params.group) {
+      const relatedLibData = await getLibraryData({
+        library: slug,
+        ref: 'latest',
+        ...libraryParams
+      })
+      if (
+        relatedLibData?.content.id !== libraryData.content.id &&
+        !relatedLibData?.content?.noIndex
+      ) {
+        relatedLibs.push(relatedLibData)
+      }
+    }
+  }
```

Comment:
My bad, we need to assert that the current library has a group before finding the related libs, otherwise it's matching libraries that have `group: undefined`, example: http://localhost:3000/libraries/ibm-security
so just need to wrap the hole thing in a `if(libraryData.params.group)`; probably need to do the same thing in the otherAssetFrameworks logic too 👉 👈 
```suggestion
if(libraryData.params.group){
  for (const [slug, libraryParams] of Object.entries(libraryAllowList)) {
    if (libraryParams.group === libraryData.params.group) {
      const relatedLibData = await getLibraryData({
        library: slug,
        ref: 'latest',
        ...libraryParams
      })
      if (
        relatedLibData?.content.id !== libraryData.content.id &&
        !relatedLibData?.content?.noIndex
      ) {
        relatedLibs.push(relatedLibData)
      }
    }
  }
  }
```